app/scripts/controllers/push-platform-notifications/firebase/lib/firebase-messaging-sw.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable */
```

Comment:
A few questions:
- why does this file need to be minified?
- is it copied from somewhere? if so where? and then does a license need to be included?
- can we include a comment at the top of this file explaining what it is?

Reply:
Same for the below file

Reply:
Ok, see the comment above.


The library is minified because it is published that way here https://www.gstatic.com/firebasejs/10.8.0/firebase-messaging-sw.js.

In any case, I have proceeded to expand it.

Reply:
Sure. See the comment  above.

The library is minified because it is published that way here [https://www.gstatic.com/firebasejs/10.8.0/firebase-messaging-sw.js](https://www.gstatic.com/firebasejs/10.8.0/firebase-messaging-sw.js).

In any case, I have proceeded to expand it.